src/utils/FetchConfig.ts

```diff
@@ -34,15 +34,18 @@ export async function FetchConfig (): Promise<any> {
   const dashboardUrl: string = import.meta.env.VUE_APP_DASHBOARD_URL
   sessionStorage.setItem('DASHBOARD_URL', dashboardUrl)
 
-  const legalApiUrl: string = (import.meta.env.VUE_APP_LEGAL_API_URL + import.meta.env.VUE_APP_LEGAL_API_VERSION_2 + '/')
+  const legalApiUrl: string = (
+    import.meta.env.VUE_APP_LEGAL_API_URL + import.meta.env.VUE_APP_LEGAL_API_VERSION_2 + '/'
+  )
```

Comment:
Line was too long.

Reply:
We probably need to fix lint at some point to make it not run the project if there was a lint error. I remember I created a ticket for that back when I did the Vite work but no idea when will we be able to get to that ticket.

Reply:
I think all we need to do is change lint warnings to errors. But this is a problem for another day. (Hope you agree!)

Reply:
Oh yes, for sure Sev! I also found the ticket: https://github.com/bcgov/entity/issues/17320

A problem for another day 🙌